Koha/Plugin/EDS/js/EDSScript.js

```diff
@@ -8,7 +8,7 @@ var edsSelectedKnownItem = "";
 var browseNextPage = "";
```

Comment:
This file includes multiple variables that are specific to the settings of a specific customer.  We should probably clear these out since the .tt file is going to build this file on save of the configuration, or potentially remove this file entirely and let the EDScript.tt file build it from nothing.